subprojects/docs/docs.gradle

```diff
@@ -120,7 +120,8 @@ tasks.withType(CacheableAsciidoctorTask) {
         }
     }
 
-    attributes website: 'http://www.gradle.org'
+    attributes website: 'http://www.gradle.org',
+               javaApi: 'https://docs.oracle.com/javase/8/docs/api'
```

Comment:
We might want to reference the Java 7 docs here as we still supporting it as the minimal JDK version.

Reply:
I'm fine with that. One of the reasons I made it a document attribute :)

Reply:
- [ ] Reference Java 7 API docs rather than Java 8